client/context/context.go

```diff
@@ -332,6 +332,7 @@ func (ctx CLIContext) PrintOutput(toPrint interface{}) error {
 // GetFromFields returns a from account address and Keybase name given either
 // an address or key name. If genOnly is true, only a valid Bech32 cosmos
 // address is returned.
+// XXX What does genOnly mean?  Why is it called that?
```

Comment:
`genOnly` == GenerateOnly. The godoc states its use. I'd advise against adding a `XXX` and actually creating an issue to revise the API.